k0799343/quiz1.py

```diff
@@ -0,0 +1,38 @@
+############################
+#
+# Please complete this quiz
+#
+
+# declare an int variable named total with the value of 0
+total = 0
+# declare an array named numbers with all numbers 0 - 20
+numbers = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
+# write a for loop that processes each number
+
+for num in numbers:
+    if num == (2,4,6,8,10,12,14,16,18,20):
```

Comment:
If you were to write something like
```
x = (2,4,5)
```

I would say that `x` is a 3 element `tuple`

So what is asked in this question is _is num a tuple with the numbers 2,4,6,8,10,12,14,16,18,20 in it_